### ISSUE TITLE
[Issue]: Unable to set positionTicks for an item via API_KEY

### ISSUE THREAD

**Issue by luigi311** [jellyfin/jellyfin#9614]
### Please describe your bug

I am unable to set the positionTicks for an item for a user via the api key from the server. Using a delete request as following /Users/{userId}/PlayingItems/{itemId}?positionTicks={positionTicks} via a user token works but using that same request with a api key from the server does not. It also doesnt seem like theres any way to generate a user token via the api key so it doesnt look like theres a way to handle it that way either.

The logs report that the playback was stopped for the correct item at the correct time but since its with a api key it doesnt seem to handle it correctly even though the request has the user id in it so it should know who this belongs to.

### Jellyfin Version

10.8.0

### Environment

```markdown
- OS: Unraid
- Virtualization: Docker
- Clients: Python Script
- Browser: N/A
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy: traefik
- Base URL:
- Networking: bridge
- Storage: local
```

### Jellyfin logs

```shell
[INF] [85] Emby.Server.Implementations.Session.SessionManager: Playback stopped reported by app "JellyPlex" "5.0.0" playing "Missing". Stopped at "1800000" ms
[INF] [137] Emby.Server.Implementations.Session.SessionManager: Playback stopped reported by app "JellyPlex" "5.0.0" playing "Just a Regular Irregular". Stopped at "900000" ms
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by luigi311** [comment]
This is still an issuee

**Comment by luigi311** [comment]
this is still an issue

**Comment by chuckgmthompson** [comment]
+1

**Comment by luigi311** [comment]
This was resolved in 10.9
